libifupdown/lifecycle.c

```diff
@@ -72,15 +72,19 @@ handle_executors_for_phase(const struct lif_execute_opts *opts, char *const envp
 	if (up)
 	{
 		LIF_DICT_FOREACH(iter, &iface->vars) {
-			if (!handle_single_executor_for_phase(iter->data, opts, envp, phase, iface->ifname))
+			if (!handle_single_executor_for_phase(iter->data, opts, envp, phase, iface->ifname)) {
 				ret = false;
+				break;
```

Comment:
We could also just change this to `return false` and get rid of `ret`.

Reply:
Yeah, that sounds like a good idea to me, if we `break` the `ret` doesn't add much value anymore.

Reply:
Done, thanks!